### PR TITLE
[v0.6] Adjust memory allocation for gateway serverless functions

### PR DESCRIPTION
**Pull request by amarthadan**
Backporting changes from https://github.com/api3dao/airnode/pull/1155 to v0.6

### CODE REVIEW

**Review by Siegrift — APPROVED** [review]
I think both AWS and GCP should be updated (like implemented in this PR, but wait for Buraks answer in Slack before merging).

Btw. can you fix the documentation link? It seems there is a broken validator link.

**Comment by amarthadan** [comment]
> Btw. can you fix the documentation link? It seems there is a broken validator link.

I was ignoring that as I thought `0.6.2` will be the last `0.6.x` release :sweat_smile: I'll fix it if `0.6.5` will become a thing :see_no_evil:

**Comment by Siegrift** [comment]
Why not fix it now? :D

**Pull request merged**
